README.md

```diff
@@ -49,38 +49,49 @@ In order to run this simulation on your own gene expression data the following s
 First we need to set up your local repository: 
 1. Clone the ```simulate-expression-compendia``` repository
 2. Set up conda environment using the command above
-3. Create a new analysis folder in the main directory (i.e. "NAME_analysis")
-4. Within your analysis folder add the same folders as found in ```Pseudomonas_analysis``` (i.e. experiment_lvl_sim/, sample_lvl_sim/, data/, logs/, models/)
-5. Copy contents of ```Pseudomonas_analysis/Pa_sample_lvl_sim/``` into your respective folder.  
-6. Copy contents of ```Pseudomonas_analysis/Pa_experiment_lvl_sim/``` into your respective folder.  
+3. Create a new analysis folder in the main directory. This is equivalent to the ```Pseudomonas``` directory
+4. Within your analysis folder create ```data/``` directory and ```input/```, ```metadata/``` subdirectories
```

Comment:
Specifically the ones in step 4, to clarify

Reply:
Technically the script can create those directories but then the script will halt because the data isn't found.  I think these are the minimum directories that the user will need to add to run the scripts